ebos/eclwriter.hh

```diff
@@ -266,7 +266,7 @@ public:
         std::vector<Opm::RestartKey> extraKeys = {{"OPMEXTRA", Opm::UnitSystem::measure::identity, false},
                                                   {"THRESHPR", Opm::UnitSystem::measure::pressure, inputThpres.active()}};
 
-        unsigned episodeIdx = simulator_.episodeIndex();
+        unsigned episodeIdx = simulator_.episodeIndex()+1;
```

Comment:
the variable should probably renamed to something akin `reportStepNum` to avoid confusion...

Reply:
Ok. I will grap it directly from the initconfig as well to avoid problems.